corehq/apps/users/models.py

```diff
@@ -1623,13 +1630,34 @@ def sql_location(self):
                 pass
         return None
 
+    @property
+    def sql_locations(self):
+        from corehq.apps.locations.models import SQLLocation
+        if self.assigned_location_ids:
+            try:
+                return SQLLocation.objects.filter(location_id__in=self.assigned_location_ids)
+            except SQLLocation.DoesNotExist:
```

Comment:
@sravfeyn 
Noticed this when looking at https://github.com/dimagi/commcare-hq/pull/14019#discussion_r89429985
I don't think `filter` ever throws a `DoesNotExist`, but instead returns an empty queryset.